tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -0,0 +1,96 @@
+import pytorch_lightning as pl
+import torch
+from torch import nn
```

Comment:
nit-picking
```suggestion
import torch.nn as nn
```
to align with `import torch.nn.functional as F`.

Reply:
Sorry, maybe I missed this but is this a style rule in Optuna? I find it quite unusual to use aliases without changing the name of the (sub)module.

Reply:
Not a style rule in Optuna and that makes sense.

(Unfortunately) PyTorch examples and I do that kind of aliasing...
https://github.com/pytorch/examples/blob/ee964a2eeb41e1712fe719b83645c79bcbd0ba1a/mnist/main.py#L1-L7

Reply:
Thanks for the reference. It looks aligned and proper, I personally find it okay as long as it's agreed upon but would "by default" write it this way.